### DISCUSSION TITLE
Remote control

### DISCUSSION THREAD

**Discussion by Mejdunn** [ChurchApps/ChurchAppsSupport#322] [FreeShow] [answered]
Dear all,
I would kindly ask, if remote access for Android phones or remote via browser will be available in the near future
 - for finding and adding lyrics and scripts
 - and to control projector output by selecting 4th verse of the song and change themes, for example?

Sincerely

M

**Comment by vassbo** [comment, marked as answer]
There is remote control for any device via a web browser. And you can find songs and enable e.g. the 4th verse.

But maybe someday I will make it so you can edit the layout/text as well.

**Comment by christopherthake** [comment]
Hi there, from a Church based in London UK. I came across this app thanks to one of my Tech Team. Any plans for docker or any thoughts on how to run this containerised? I see you have a linux package, wondered if support for RPi would be coming?

Thanks,
Christopher

**Comment by vassbo** [comment]
Nice! No plans for docker at this moment.
I found this if it helps: https://medium.com/@SaravSun/running-gui-applications-inside-docker-containers-83d65c0db110
You should be able to install the deb package on RPi.

**Reply by christopherthake** [reply]
Thanks Vassbo, I will look into this and let you know how I get on
